orangecontrib/text/widgets/owtweetprofiler.py

```diff
@@ -1,15 +1,34 @@
-from AnyQt.QtWidgets import QApplication, QGridLayout, QLabel, QGroupBox, QHBoxLayout
+import numpy as np
+
+from AnyQt.QtWidgets import QApplication, QGridLayout, QLabel, QGroupBox, \
+    QHBoxLayout, QPushButton, QStyle
 
 from Orange.data import StringVariable
 from Orange.widgets import gui
 from Orange.widgets.settings import Setting
 from Orange.widgets.utils.itemmodels import VariableListModel
+from Orange.widgets.utils.concurrent import ConcurrentWidgetMixin
 from Orange.widgets.widget import OWWidget, Msg, Input, Output
 from orangecontrib.text.tweet_profiler import TweetProfiler
 from orangecontrib.text.corpus import Corpus
 
 
-class OWTweetProfiler(OWWidget):
+def run_profiler(profiler, corpus, meta_var, model_name, output_mode, state):
+
+    ticks = iter(np.linspace(0., 100.,
+                             np.ceil(len(corpus) / profiler.BATCH_SIZE)))
```

Comment:
I get an error here since `np.ceil(len(corpus) / profiler.BATCH_SIZE)` does not return an integer but float and linspace seems to require an integer as a third argument.

Reply:
Seems that my numpy version is older and still supports float argument. It's fixed now.